editor/src/components/canvas/controls/comment-popup.tsx

```diff
@@ -493,7 +494,7 @@ const NewCommentPopup = React.memo((props: NewCommentPopupProps) => {
   )
 
   return (
-    <>
+    <div>
```

Comment:
what was the reason for turning this fragment into a div? do we still need the div at all?